Formula/mat2.rb

```diff
@@ -39,11 +39,11 @@ def install
 
     resources.each do |r|
       r.stage do
-        system python, *Language::Python.setup_install_args(libexec/"vendor", python), "--install-data=#{prefix}"
+        system python, "-m", "pip", "install", *std_pip_args(prefix: libexec/"vendor"), "."
       end
     end
 
-    system python, *Language::Python.setup_install_args(prefix, python)
+    system python, "-m", "pip", "install", *std_pip_args, "."
     bin.env_script_all_files(libexec/"bin", PYTHONPATH: ENV["PYTHONPATH"])
```

Comment:
This will break the `:all` bottles.

Reply:
we might already have the similar issue when migrating builds like fb-client, https://github.com/Homebrew/homebrew-core/pull/138014 cc @branchvincent

Reply:
`fb-client` wasn't an `:all` bottle though? 

If I've migrated one, it was on accident. The issue is from recording the build's cwd https://github.com/Homebrew/homebrew-core/pull/136571#issuecomment-1635310496. My plan is to revisit these soon, one possible solution takes two `pip` invocations (a `pip wheel` to build it and a `pip install --find-links` to install it) but want to try to find a cleaner solution